release.sh

```diff
@@ -44,20 +44,25 @@ SAT_REPO_TYPE="unstable"
 SEMANTIC_VERSION_ONLY=${RELEASE_VERSION%%-*}
 MAJOR_MINOR_VERSION=${SEMANTIC_VERSION_ONLY%.*}
 
-# Substitute SAT version
+DISABLE_GPG_CHECK=yes
+if [[ "${SAT_REPO_TYPE}" == "stable" ]]; then
+    DISABLE_GPG_CHECK=no
+fi
+
+# Substitute SAT variables
 source "${ROOTDIR}/component_versions.sh"
-for f in "${ROOTDIR}/docker/index.yaml" "${ROOTDIR}/cray-product-catalog/sat.yaml" "${ROOTDIR}/install.sh" \
-    "${ROOTDIR}/rpm/sle-15sp2/index.yaml" "${ROOTDIR}/helm/index.yaml" "${ROOTDIR}/manifests/sat.yaml"
+for f in "${ROOTDIR}/docker/index.yaml" "${ROOTDIR}/cray-product-catalog/sat-component-versions.yaml" \
```

Comment:
Definitely not required here, but as I was developing the hpc-shasta-software-recipe repo, I thought it would be nicer to define a bash array of these files, and make them all paths relative to `${ROOTDIR}` and then iterate over the contents of the array. Seems like it would be a little more readable and less error-prone.